testing/ports.csv

```diff
@@ -6,4 +6,6 @@ python/MDSplus/tests/connection-thick, 8013
 python/MDSplus/tests/connection-tcp, 8014
 python/MDSplus/tests/connection-write, 8015
 python/MDSplus/tests/dcl-dispatcher, 8016-8017
-python/MDSplus/tests/dcl-timeout, 8018
\ No newline at end of file
+python/MDSplus/tests/dcl-timeout, 8018
+java/mdsobjects/test/MdsRemoteEventTest, 8019
```

Comment:
I really appreciate you allocating the ports! This could be easily looked over, and it's a big help to keep things organized